tests/test_scalars.py

```diff
@@ -752,7 +793,7 @@ def test_period_add_subtract() -> None:
     as_period = pd.Period("2012-1-1", freq="D")
     scale = 24 * 60 * 60 * 10**9
     as_td_series = pd.Series(pd.timedelta_range(scale, scale, freq="D"))
-    check(assert_type(as_td_series, TimedeltaSeries), pd.Series)
+    check(assert_type(as_td_series, TimedeltaSeries), pd.Series, pd.Timedelta)
     as_period_series = pd.Series(as_period_index)
     check(assert_type(as_period_series, PeriodSeries), pd.Series)
```

Comment:
add `pd.Period` as part of the argument to `check` here

Reply:
Added